overlord/snapstate/export_test.go

```diff
@@ -54,5 +54,5 @@ func (m *SnapManager) AddForeignTaskHandlers() {
 
 func ChangeRetrieveInfo(retrieve func(name string, si *snap.SideInfo) (*snap.Info, error)) func() {
 	retrieveInfo = retrieve
-	return func() { retrieveInfo = retrieveInfoImpl }
+	return func() { retrieveInfo = snap.InfoWithSide }
```

Comment:
`MockReadInfo`, `readInfo`, `snap.ReadInfo`

Seem like a nice chance to make it all more consistent.